spaces/S000109/properties/P000049.md

```diff
@@ -0,0 +1,10 @@
+---
+space: S000109
+property: P000049
+value: true
+refs:
+- mathse: 3769214
+  name: Dense Subspace of Extremally Disconnected Space is Extremally Disconnected
+---
+
+{S109} is dense in {S108} and {S108|P49}, and a dense subspace of an extremally disconnected space is extremally disconnected (see {{mathse:3769214}}).
```

Comment:
```suggestion
---

{S109} is dense in {S108} and {S108|P49}, and a dense subspace of an extremally disconnected space is extremally disconnected.
```
No need for the reference anymore, as it's now part of the meta-properties section for extremally disconnected on the main branch (from #1206).